resources/tiers.md

```diff
@@ -0,0 +1,84 @@
+# SDK tiers
+
+The objective of Meilisearch's Integrations Team is to **ease Meilisearch usage to as many developers as possible**.
+
+Given this statement, **tiers** are nothing more than a __group of SDKs/integrations__ where the integrations team can fairly divide their attention. Unfortunately is not possible to keep all the SDKs 100% updated against the new Meilisearch releases all the time, the Integrations Team is a polyglot small team that maintains 30+ different integrations and tools, so it is humanly impossible to consistently meet the Meilisearch quality standards.
+
+The ultimate goal of introducing tiers is to allow the team to have more time for new projects/integrations so that more time can be invested and more users will be impacted.
+
+It's been using [data collected here](https://docs.meilisearch.com/learn/what_is_meilisearch/telemetry.html) and data like GitHub stars, forks, and watchers to measure the impact of an SDK.
+
+It is worth remembering that this priority is just a concept introduced by the team where it aims to give a decision weight before
+we invest time in some issues of some SDK in question.
+
+:warning: We will keep all officially maintained SDKs **working**. :warning:
```

Comment:
```suggestion
:warning: We keep all officially maintained SDKs **working**. :warning:
```

What do you mean with working? We do not leave bugs behind?

Reply:
Not really. Let me give you a hypothetical example of it:

Let's take my scope, for example, if a user reports an error in the Dart like this https://github.com/meilisearch/meilisearch-dart/issues/242.  
I'll not leave them alone to work on a `feature request` of PHP.

Is it clear?

Reply:
Okey! Yes it is :)  We fix critical bugs